src/main/java/jenkins/scm/impl/subversion/SubversionSCMSource.java

```diff
@@ -231,7 +231,7 @@ protected void retrieve(@CheckForNull SCMSourceCriteria criteria,
                             @NonNull final SCMHeadObserver observer,
                             @CheckForNull SCMHeadEvent<?> event,
                             @NonNull TaskListener listener)
-            throws IOException {
+            throws IOException, InterruptedException {
```

Comment:
If the body does not throw it then it may be removed. It is not a binary compatibility issue since the JVM does not care about checked exceptions.